diff -Naur qt/qtbase/src/gui/image/qjpeghandler.pri Qt5--qtbase/qtbase/src/gui/image/qjpeghandler.pri
--- qt/qtbase/src/gui/image/qjpeghandler.pri	2016-12-01 09:17:04.000000000 +0100
+++ Qt5--qtbase/qtbase/src/gui/image/qjpeghandler.pri	2017-11-17 08:44:23.415090100 +0100
@@ -3,7 +3,7 @@
 SOURCES += $$PWD/qjpeghandler.cpp
 contains(QT_CONFIG, system-jpeg) {
     msvc: \
-        LIBS += libjpeg.lib
+        LIBS += jpeg.lib
     else: \
         LIBS += -ljpeg
 } else {
